src/intl/messages.js

```diff
@@ -167,6 +167,7 @@ export const messages: { [messageId: string]: MessageType } = {
   createVmWizardStepTitleNetwork: 'Networking',
   createVmWizardStepTitleReview: 'Review',
   createVmWizardStepTitleStorage: 'Storage',
+  createVmWizardStorageDiskNameRules: 'Disk name must be 40 or less alphanumeric characters or "-_."',
```

Comment:
This message is not accurate, please check that you can create a disk name longer than 40 chars. Please rephrase it to be the same as on webadmin:
Disk name can contain only alphanumeric, '.', '_' or '-' characters.

Reply:
![image](https://user-images.githubusercontent.com/64131213/103477451-b4ad0280-4dc7-11eb-9bc4-2c9a8e0f085a.png)